tests/lesson/lesson-info-tests.js

```diff
@@ -0,0 +1,55 @@
+const utils = require("../utils");
+const apiGET = utils.apiGET;
+const setup = require("../setup");
+const seedData = setup.seedData;
+
+function lessonInfoTests() {
+    it("getting verse info linked to a lesson", async () => {
+        let verseWord = seedData.VerseWord
+        let arabicWord = seedData.ArabicWord
+        let verses = seedData.Verse
+        let tafsirs = seedData.Tafsir
+        let reflections = seedData.Reflection
+
+        let result = await apiGET("/lesson/2/verses")
+        expect(result.data.data.lessonInfo.length).toEqual(2)
+
+        checkVerseMatch(result.data.data.lessonInfo[0].verse[0], verses[0])
+        checkVerseMatch(result.data.data.lessonInfo[1].verse[0], verses[1])
+
+        checkReflectionMatch(result.data.data.lessonInfo[0].reflections[0], reflections[0])
+        checkReflectionMatch(result.data.data.lessonInfo[0].reflections[1], reflections[1])
+        checkReflectionMatch(result.data.data.lessonInfo[1].reflections[0], reflections[2])
```

Comment:
Just curious why we have three checks here and not for tafsirs or words? What's the least required number of checks we can make?

Reply:
we can check just one for each, and make sure the number of entries returned is as expected